### DISCUSSION TITLE
Datapack for epic fight

### DISCUSSION THREAD

**Discussion by arslan1510** [TerraFirmaGreg-Team/TFG-Modpack-1.20.x#1040] [Show and tell]
### Version modpack Used

1.20.x Ver.0.7.12

### Related Problem

I made a epic fight datapack for TFC, ik epic fight is more action rpg theme with its skills and all but when its rpg elements kept at min i.e. only provide user with step or dodge ability and disabling all abilities and epic fight items, then it really suits the pack well, it greatly enhances the combat we have, more like QoL change.

### Your Solution

Here is the datapack and related KubeJS scripts to disable items for epic fight.
[TFG-EPDatapack.zip](https://github.com/user-attachments/files/16659835/TFG-EPDatapack.zip)
[kubejs.zip](https://github.com/user-attachments/files/16659864/kubejs.zip)

**Comment by RedBlockCreator** [comment]
What version of Epic Fight, and does with work with TFG .9+?

**Reply by arslan1510** [reply]
it should if i remember correctly the new epic fight didnt changed the way it patches weapon types
